fastcrypto-zkp/src/bn254/zk_login.rs

```diff
@@ -341,7 +317,7 @@ impl AuxInputs {
         let mut poseidon = PoseidonWrapper::new(11);
         let jwt_sha2_hash_0 = Bn254Fr::from_str(&self.jwt_sha2_hash[0]).unwrap();
```

Comment:
are all those unwraps are safe? maybe use expect("reason") / document

Reply:
leaving as todo for now